data/campaigns/Descent_Into_Darkness/scenarios/07_A_Small_Favor.cfg

```diff
@@ -187,33 +234,155 @@
-    {PLACE_IMAGE items/altar.png 12 5}
+    {PLACE_IMAGE scenery/monolith1.png 13 6}
+    {PLACE_IMAGE scenery/monolith1.png 10 6}
+    {PLACE_IMAGE scenery/signpost.png 15 16}
+    {PLACE_IMAGE scenery/signpost.png 11 12}
+    {PLACE_IMAGE scenery/temple1.png 10 18}
+    {PLACE_IMAGE scenery/tent-fancy-red.png 9 15}
+    {PLACE_IMAGE items/dragonstatue.png 12 14}
+
+    {PLACE_IMAGE "items/dummy.png~RC(magenta>green)" 7 12}
+    {PLACE_IMAGE "items/dummy.png~RC(magenta>green)" 4 11}
+    {PLACE_IMAGE items/armor.png 4 12}
+    {PLACE_IMAGE items/armor.png 4 16}
+    {PLACE_IMAGE items/armor.png 6 16}
+    {PLACE_IMAGE items/flower2.png 13 7}
+    {PLACE_IMAGE items/flower1.png 16 8}
+    {PLACE_IMAGE items/flower3.png 12 7}
+    {PLACE_IMAGE items/flower4.png 17 8}
 
     # Manor
     {PLACE_IMAGE scenery/trapdoor-closed.png 3 6}
-    {PLACE_IMAGE items/cloak-green.png 5 7}
+    {PLACE_IMAGE items/cloak-green.png 2 6}
     {PLACE_IMAGE items/ornate1.png 5 5}
     {PLACE_IMAGE items/potion-blue.png 4 5}
     {PLACE_IMAGE items/potion-red.png 6 5}
+    {PLACE_IMAGE items/altar.png 3 7}
+
+#define WATCHMAN_PATROL_MOVEMENT
+    [set_variable]
+        name=turn_mod
+        value=$($turn_number % 5)
+    [/set_variable]
+
+    [switch]
+        variable=turn_mod
+        [case]
+            value=1,0
+            [move_unit]
+                id=Guard3
+                side=2
+                to_x,to_y=5,22
+            [/move_unit]
+            [move_unit]
+                id=Guard2
+                side=2
+                to_x,to_y=4,21
+            [/move_unit]
+            [move_unit]
+                id=Guard1
+                side=2
+                to_x,to_y=4,20
+            [/move_unit]
+
+            [move_unit]
+                id=Guard6
+                side=2
+                to_x,to_y=18,7
+            [/move_unit]
+            [move_unit]
+                id=Guard5
+                side=2
+                to_x,to_y=16,6
+            [/move_unit]
+            [move_unit]
+                id=Guard4
+                side=2
+                to_x,to_y=14,5
+            [/move_unit]
+        [/case]
+        [case]
+            value=2,4
+            [move_unit]
+                id=Guard3
+                side=2
+                to_x,to_y=10,24
+            [/move_unit]
+            [move_unit]
+                id=Guard2
+                side=2
+                to_x,to_y=10,23
+            [/move_unit]
+            [move_unit]
+                id=Guard1
+                side=2
+                to_x,to_y=9,24
+            [/move_unit]
+
+            [move_unit]
+                id=Guard6
+                side=2
+                to_x,to_y=22,11
+            [/move_unit]
+            [move_unit]
+                id=Guard5
+                side=2
+                to_x,to_y=21,11
+            [/move_unit]
+            [move_unit]
+                id=Guard4
+                side=2
+                to_x,to_y=20,9
```

Comment:
The guards are shown as having only 3 mp, but on AI turn 9 Guard4 moves from 21,15 to 20,9, which should need 6 mp.

Reply:
Should be okay now, I made some fixes.